doc/files/package.json.md

```diff
@@ -393,6 +393,15 @@ shortcut syntax you use for `npm install`:
 
     "repository": "gitlab:user/repo"
 
+If your package is part of a monorepo, you should specify the directory in which
```

Comment:
Just because the package is not in the repository, doesn't mean it's in a monorepo - but it should still use the `directory` field.
Example: https://github.com/badges/shields/tree/master/gh-badges is the only npm package in the repository (so it's not a monorepo), but not in the root. This makes it sound like it should not use the `directory` field.